chore(contrib): Adding Bluesky, replace links to contributing.md (#39833)
mdn/content: files/en-us/mdn/community/roles_teams/index.md

To be a contributor, you must follow:

- [Mozilla's code of conduct](https://www.mozilla.org/en-US/about/governance/policies/participation/)

> - Contribution guidelines per repository. You will find one in a `CONTRIBUTING.md` file in the project root if there is any project-specific hints or guidelines.

Reviewer: ```suggestion
- The specific guidelines for each repository. If there are any project-specific hints or guidelines, you will find them in a `CONTRIBUTING.md` file in the project repository root.
```
